### PR TITLE
update examples main README

### PR DESCRIPTION
**Pull request by t-fine**
Signed-off-by: Troy Fine <Troy.Fine@ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,48 @@
...
 
-Edge examples specific to the Watson IoT Platform are found in:
- - edge/wiotp
+## Documentation
+**Open Horizon documentation repository coming soon!** For the time being, you can learn more about [Open Horizon here](https://www.ibm.com/support/knowledgecenter/SSFKVV_4.2/kc_welcome_containers.html).
+
+## Management Hub Installation
+Before you can publish and use any of the services in this repository, you must first deploy your own Horizon Management Hub. This can be done with one simple command using the `deploy-mgmt-hub.sh` script located in the [devops repository](https://github.com/open-horizon/devops/tree/master/mgmt-hub#horizon-management-hub). This will give you with a management hub with several services, policies and patterns published in the exchange. 
+
+## Register an Edge Node with your Mangement Hub
```
Comment (clementkng) [inline README.md:11]: nit: Management Hub

File: README.md [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
+Currently there is only one example service in this repository that is designed to run on an edge cluster and that is the [nginx-operator](edge/services/nginx-operator).
+
+# Example Services 
+During the management hub installation, several services should have been published into the exchange automatically. The following three command will list the services, patterns, and deployment policies available in your exchange:
+```
+hzn exchange service list IBM/
+hzn exchange pattern list
+hzn exchange deployment listpolicy 
+```
+**Note:** The above commands assume you have `HZN_ORG_ID` set to the created user org (default is `mycluster`), and your `HZN_EXCHANGE_USER_AUTH` set accordingly.
```
Comment (clementkng) [inline README.md:38]: The org default is sometimes `myorg` as well.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,48 @@
...
+
+You can find a list of available edge services in this repository located in the [edge/services](edge/services) directory. For the most part, each of the services are boken up into micro-services designed to accomplish one specific task. This makes them easier to incorporate into a wide variety of "top-level" services.
```
Comment (clementkng) [inline README.md:40]: nit: broken

File: README.md [hunk 4]
```diff
@@ -1,12 +1,48 @@
...
+hzn exchange pattern list
+hzn exchange deployment listpolicy 
+```
+**Note:** The above commands assume you have `HZN_ORG_ID` set to the created user org (default is `mycluster`), and your `HZN_EXCHANGE_USER_AUTH` set accordingly.
+
+You can find a list of available edge services in this repository located in the [edge/services](edge/services) directory. For the most part, each of the services are boken up into micro-services designed to accomplish one specific task. This makes them easier to incorporate into a wide variety of "top-level" services. 
+
+A good example of a "top-level" service is [cpu2evtstreams](edge/evtstreams/cpu2evtstreams), which has two dependent services ([cpu_percent](edge/services/cpu_percent), and [gps](edge/services/gps)). It uses these two micro-services to gather information about the edge node it is running on and sends it to an instance of IBM Event Streams using `kafkacat`.
+
+Edge examples specific to the Watson IoT Platform are found in [edge/wiotp](edge/wiotp). These examples are not being maintained.
```
Comment (clementkng) [inline README.md:44]: This isn't specific to this review, but should we archive or remove the Watson IoT Platform examples at some point?

**Review by clementkng — CHANGES_REQUESTED** [review]
Looks good so far!

File: README.md (hunk 2 above)
Comment (t-fine) [reply README.md:38]: Ah, good point! I forget that's the default when running the all-in-1 script. I'll have to add that in

File: README.md (hunk 1 above)
Comment (t-fine) [reply README.md:11]: Good catch

File: README.md (hunk 4 above)
Comment (t-fine) [reply README.md:44]: I would certainly like to do something like that. I'd like to get Bruce's input on that as well

File: README.md (hunk 3 above)
Comment (t-fine) [reply README.md:40]: Will fix

File: README.md [hunk 5]
```diff
@@ -1,12 +1,48 @@
...
+
+## Register an Edge Node with your Mangement Hub
+In order to deploy a service to an edge node it must first be registered with a management hub. The `agent-install.sh` script is a fast and easy way to register an edge node with a management hub, more information can be found in the [open-horizon/anax](https://github.com/open-horizon/anax/tree/master/agent-install#edge-node-agent-install) repository. Edge nodes can be either a device or a cluster. Open Horizon edge cluster capability helps you manage and deploy workloads from a management hub cluster to remote instances of OpenShift® Container Platform or other Kubernetes-based clusters. 
+
+Typically, **edge devices** have a prescriptive purpose, provide (often limited) compute capabilities, and are located near or at the data source. Currently supported edge device OS and architectures:
```
Comment (bmpotter) [inline README.md:14]: We also support red hat derivatives, see https://www-03preprod.ibm.com/support/knowledgecenter/SSFKVV_4.2/installing/adding_devices.html

File: README.md [hunk 6]
```diff
@@ -1,12 +1,48 @@
...
+* [K3s - Lightweight Kubernetes](https://rancher.com/docs/k3s/latest/en/)
+* [MicroK8s](https://microk8s.io/docs) on Ubuntu 18.04 (for development and test, not recommended for production)
+
+Currently there is only one example service in this repository that is designed to run on an edge cluster and that is the [nginx-operator](edge/services/nginx-operator).
+
+# Example Services 
+During the management hub installation, several services should have been published into the exchange automatically. The following three command will list the services, patterns, and deployment policies available in your exchange:
+```
+hzn exchange service list IBM/
+hzn exchange pattern list
```
Comment (bmpotter) [inline README.md:35]: The example patterns are in the IBM org, so this cmd should be `hzn exchange pattern list IBM/`

File: README.md (hunk 4 above)
Comment (bmpotter) [reply README.md:44]: Yes, i agree.

**Review by bmpotter — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (t-fine) [reply README.md:14]: Updating this now

File: README.md (hunk 6 above)
Comment (t-fine) [reply README.md:35]: added

File: README.md (hunk 4 above)
Comment (t-fine) [reply README.md:44]: Perfect, I'll take care of this in the very near future

**Review by bmpotter — APPROVED** [review]

**Pull request merged**